### DISCUSSION TITLE
Bug with label

### DISCUSSION THREAD

**Discussion by juandiii** [gpbl/react-day-picker#2066] [Support]
## Description

I found a bug when I place the label inside the 'Day Picker', and then when I go to the browser and hover over the 'Day Picker', it displays the back button (hover) to go to the previous month.

CodeSandbox: https://codesandbox.io/p/sandbox/react-day-picker-with-label-5lc4cg 

## Expected Behavior

It shouldn't display the back button (hover) to go to the previous month unless a specific functionality is clicked to change the month.

## Actual Behavior

I found a bug when I place the label inside the 'Day Picker', and then when I go to the browser and hover over the 'Day Picker', it displays the back button (hover) to go to the previous month. 

Or when you hover over and click outside of the calendar, meaning the edges

**Comment by gpbl** [comment]
@juandiii thanks for the report! You should not wrap DayPicker with a `<label />`. 

See docs about "Content model" for the label element: https://html.spec.whatwg.org/multipage/forms.html#the-label-element
